weaviate/collections/aggregations/hybrid.py

```diff
@@ -25,6 +25,7 @@ async def hybrid(
         filters: Optional[_Filters] = None,
         group_by: Optional[Union[str, GroupByAggregate]] = None,
         target_vector: Optional[str] = None,
+        max_vector_distance: Optional[float] = None,
```

Comment:
Should this type be `Optional[NUMBER]` to align with that in the `.queries` methods or is this a point-of-difference between the two underlying protocols?